Week1/Homework/js-exercises/logNumber.js

```diff
@@ -0,0 +1,15 @@
+"usestrict";
+
+var numberX;
```

Comment:
Would you use `let` or `const` here?

Reply:
I have to use `let` because I change the value of numberX later in the program.